### ISSUE TITLE
Linux release

### ISSUE THREAD

**Issue by phocean** [brimsec/brim#586]
Please release a Linux version.

Flatpack packaging is probably ideal for you, to quickly reach all users regardless of the distro.

**Comment by philrz** [comment]
@phocean: Thanks for the issue. Yes, Linux support is definitely something on our to-do list, and the tip on Flatpak is helpful. In the event that distro does end up being a concern at all, I'd appreciate it if anyone who stumbles onto this issue before we release Linux support could mention their preferred distro & version here or in the [thread on our public Slack](https://brimsec.slack.com/archives/C010DR0HHMF/p1585167225002800?thread_ts=1585097486.000100&cid=C010DR0HHMF).

**Comment by phocean** [comment]
Thanks, great news!

openSUSE Tumbleweed here (rolling distro, RPM packaging / Flatpack).

**Comment by Vladontheroad** [comment]
OpenSUSE Tumbleweed here !
Thanks a lot !

**Comment by laoshaw** [comment]
ubuntu debian

**Comment by dboekhout** [comment]
ubuntu / debian

**Comment by siskojr** [comment]
We have also got requests for:
1. Kali (-> Debian)
2. Redhat

**Comment by MariaRigaki** [comment]
Ubuntu / Debian

**Comment by ethack** [comment]
Ubuntu 16.04 / 18.04

**Comment by andreagrax** [comment]
Ubuntu 18.04

**Comment by philrz** [comment]
Good news for those of you who were waiting on a Linux release! We now have `.deb` and `.rpm` packages as part of the Brim `v0.9.1` release that's linked from the [Downloads](https://www.brimsecurity.com/download/) page.

Also, as relates to the Flatpak tip that kicked off this issue, we did pursue that, but hit some problems with Zeek, so the subject is on hold while we wait for resolution there. We're tracking Flatpak via a separate issue https://github.com/brimsec/brim/issues/685. Hopefully Linux folks can get by with one of the other packages in the interim.

Please do try out the new Linux support and come see on [Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg) if you have issues, questions, feedback, etc. Thanks for using Brim!
